components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -1780,7 +1812,13 @@ bool AdsServiceImpl::ShouldShowNotifications() const {
 
 void AdsServiceImpl::CloseNotification(
     const std::string& id) {
-  display_service_->Close(NotificationHandler::Type::BRAVE_ADS, id);
+  // we might want to close Brave ads notification
+  // between browser sessions and
+  // NotificationPlatformBridgeAndroid::regenerated_notification_infos_
+  // possibly was purged already, so we need a way to identify a Brave ad.
+  std::string prefixed_id(kBraveAdsUrlPrefix);
+  prefixed_id += id;
```

Comment:
url is created here and in ad_notifications.cc

Reply:
NONCRITICAL
Covered by: brave/brave-browser#6235 (2)